store/interface.go

```diff
@@ -99,6 +99,7 @@ type Reader interface {
 	LastCertificate() *certificate.BlockCertificate
 	IsBanned(addr crypto.Address) bool
 	IsPruned() bool
+	RetentionBlocks() uint32
```

Comment:
Why don't we export "Pruningheight" instead on "RetentionBlocks"?

Reply:
Because the internal function that we call have the same name. why change it?

Reply:
RetentionBlocks is an internal configuration. We better export something that has more general information.